packages/db/client-types.ts

```diff
@@ -0,0 +1,37 @@
+// Enums copied from @prisma/client so we can use them on the client
+// Maybe we can create a script to have the enums pulled out here automatically
```

Comment:
Why are we not simply using a prisma generator to generate the files into our dedicated `@agreeto/calendar/types` package? 🤔 

Then, @agreeto/calendar/types can become a devDep for the apps that import the calendar?

Reply:
i had the types in calendar/types initially but decided to move them cause i don't see it as the calendar's responsibility to distribute prisma enums.

the api package is basically used everywhere so that seemed like the next best place.

but the enums should be generated into this file using a generator (or script whatever) and then the api package can re-export this file?

Reply:
> but the enums should be generated into this file using a generator (or script whatever) and then the api package can re-export this file?

💯  ideally prisma generator

Reply:
> 💯 ideally prisma generator

Done